ext/vulnsrc/rhel/rhel.go

```diff
@@ -184,18 +185,30 @@ func parseRHSA(ovalReader io.Reader) (vulnerabilities []database.VulnerabilityWi
 	for _, definition := range ov.Definitions {
 		pkgs := toFeatures(definition.Criteria)
 		if len(pkgs) > 0 {
+
+			// Init vulnerability
 			vulnerability := database.VulnerabilityWithAffected{
 				Vulnerability: database.Vulnerability{
-					Name:        name(definition),
-					Link:        link(definition),
```

Comment:
Maybe we could leave this code the same, but rename the functions `rhsaName(def)` and `rhsaLink(def)`.

Reply:
I agree on that, we can default to RHSA name and link.